conanfile.py

```diff
@@ -284,7 +290,15 @@ def layout(self):
         self.cpp.package.resdirs = ["resources", "plugins", "packaging", "pip_requirements"]  # pip_requirements should be the last item in the list
 
     def build(self):
-        pass
+        if self.settings.os == "Windows":
+            self.win_bash = True  # We need gettext, which requires the bash environment
```

Comment:
I know you're probably doing this because the gettext conan package relies on it, but we have been able to get it to work on Windows without, I think. (Don't ask how ... I've forgotten :-p )